pkg/urbit/noun/manage.c

```diff
@@ -496,7 +497,7 @@ _pave_south(c3_w* mem_w, c3_w siz_w, c3_w len_w)
 static void
 _pave_parts(void)
 {
-  u3R->cax.har_p = u3h_new();
+  u3R->cax.har_p = u3h_new_cache(u3_Host.ops_u.hap_w);
```

Comment:
u3_Host is a `vere` global, it really shouldn't be used in u3. But this isn't a new problem (the json tracing follows the same pattern), so it doesn't need to be fixed here.